### ISSUE TITLE
Thank you for the screen reader accessibility

### ISSUE THREAD

**Issue by whocrazy** [rmcrackan/Libation#851]
I've been meaning to write in to say thank you so much for the screen reader accessibility.
I use NVDA, nonvisual desktop access, and I am so glad there is a solution I can use to listen to audible books on the desktop, I previously used the old audible player, but that isn't supported any more, and audible have discontinued their desktop app.
Thanks again.

**Comment by rmcrackan** [comment]
You're very welcome. I built the original version (of what would become Libation Classic) using an old boring technology that I happened to be comfortable with. The downside is that it's windows only. The upside is that older mature tech has a history of slowly acquiring features like this -- I was later told that it had built-in support for screen reader accessibility.

A few years ago when we created Libation Chardonnay using newer tech which supported windows mac and linux, it was suggested that we no longer needed the Classic version. I was insistent that until the tech behind Chardonnay plays nice with screen readers, we will always maintain Classic even though it takes a lot of work to keep these 2 versions in sync with each other.

My oldest child has physical disabilities (limb-loss, not vision) so I'm keenly aware of what it means to have profoundly fewer choices when navigating the world built for someone else's body. I take it as a point of pride and responsibility to keep Libation as accessible as I can.

I'm so happy it works for you and thank you for reaching out and reinforcing this choice.
